app/artifact-cas/api/cas/v1/status_http.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go-http. DO NOT EDIT.
 // versions:
-// - protoc-gen-go-http v2.7.0
```

Comment:
this is the result of generating the API bindings with the latest version of the tools